bin/interpreter.sh

```diff
@@ -128,6 +131,8 @@ if [[ "${INTERPRETER_ID}" == "spark" ]]; then
   fi
 fi
 
+addJarInDir "${LOCAL_REPO_DIR}"
```

Comment:
ah got it.
It's a bit confusing, as "local repository" is frequently used with Maven https://maven.apache.org/guides/introduction/introduction-to-repositories.html